kernel/src/main/java/org/kframework/compile/AddImplicitComputationCell.java

```diff
@@ -63,7 +63,7 @@ public Sentence apply(Sentence s) {
         }
     }
 
-    static boolean skipSentence(Sentence s) {
+    private boolean skipSentence(Sentence s) {
```

Comment:
Previously this function is also used in `AddTopCellToRules.java`.(when adding `kore` attribute to the spec, it will skip both `AddTopCellToRules` and `AddImplicitComputationCell`). However, we only want it to skip `AddImplicitComputationCell`. So I made another skipSentence function in the `AddTopCellToRules`. There is no need to make this function static.